elements/pfe-dropdown/pfe-dropdown.ts

```diff
@@ -328,8 +351,32 @@ export class PfeDropdown extends LitElement {
     return item.shadowRoot?.querySelector(`.pfe-dropdown-item__container`) ?? null;
   }
 
+  /**
+   * Add dropdown items dynamically
+   * ```js
+   * customElements.whenDefined("pfe-dropdown").then(function() {
+   *   dropdown.addDropdownOptions(
+   *     [
+   *       {
+   *         href: "https://patternflyelements.org",
+   *         text: "Link 4",
+   *         type: "link",
+   *         disabled: false
+   *       }
+   *     ]
+   *   );
+   * });
+   * ```
+   * */
   @bound addDropdownOptions(options: PfeDropdownOption[]) {
-    this._modifyDOM(options);
+    this._modifyDOM(options, false);
+  }
+
+  /**
+   * Override the default focus method to give focus to the internal button
+   */
+  @bound focus() {
+    this.shadowRoot?.querySelector('button')?.focus();
```

Comment:
isn't this what delegatesFocus does?

Reply:
That's what I thought too but I couldn't get it to work via JS.

Reply:
Ok I'm going to give this the once over later today, see if I can get it to work 

Perhaps the issue is that the button isn't the first focusable element in the (fully projected) Shadow DOM?

Reply:
was the JS problem just in testing?

Reply:
talking this over on a call:
- the problem is primary related to testing
- our assumptions about how the `HTMLElement#focus` method relates to `:focus` styles don't obtain (https://codepen.io/bennyp/pen/JjrVOWK?editors=1100)
- we'll solve by re-doing the test to use real keyboard events via playwright via web-test-runner a la pfe-accordion tests